File: tests/moving_average_envelope_test.py
import numpy as np
from taflow import MovingAverageEnvelope
def test_moving_average_envelope_lifecycle():
    s=MovingAverageEnvelope(np.array([],float),2,.1);s.extend([1,2]);assert s.value is not None;s.reset();assert len(s)==0

File: crates/taflow-core/src/indicators/moving_average_envelope_test.rs
use super::moving_average_envelope::MovingAverageEnvelope;
#[test]
fn lifecycle() {
    let mut s = MovingAverageEnvelope::new(2, 0.1).unwrap();
    s.append(1.0);
    assert!(s.append(2.0).is_some());
    s.reset();
    assert!(s.value().is_none());
}
